torch/testing/_creation.py

```diff
@@ -103,27 +101,47 @@ def make_tensor(
                 [False, True]], device='cuda:0')
     """
 
-    def _modify_low_high(low, high, lowest, highest, default_low, default_high, dtype):
+    def modify_low_high(
+        low: Optional[float],
+        high: Optional[float],
+        *,
```

Comment:
Having `low` and `high` as positional is fine for a function called `modify_low_high`. Everything else should be keyword only to enforce readable calls. See below for the result.